helm/alfresco-content-services/tests/repository_test.yaml

```diff
@@ -0,0 +1,30 @@
+---
+suite: test repository manifest
+templates:
+  - deployment-repository.yaml
+  - config-repository.yaml
+  - secret-database.yaml
+  - secret-s3.yaml
+  - config-dev-log4j-properties.yaml
+tests:
+  - it: should have basic metadata in place
+    values: &testvalues
+      - values/test_values.yaml
+    asserts:
+      - equal:
+          path: metadata.name
+          value: RELEASE-NAME-alfresco-cs-repository
+      - equal:
+          path: spec.template.metadata.annotations.checksum/config
+          value: 65b89491b06b9ccd8146424a8eab9d7a9d109a18db722e1276f1223ee37fd47c
+  - it: should have basic metadata in place
+    values: *testvalues
+    asserts:
+      - equal:
```

Comment:
IIRC this is already asserted in the alfresco-common suite

Reply:
it's more to have a very basic assertion in every test to make sure that suite is setup correctly, we could remove all of them once we have more tests